doc/emacs/macport.texi

```diff
@@ -540,12 +540,18 @@ title/tool/tab bars look slightly colored with the value of the
 color) if the frame is focused and not in fullscreen.
 
 @vindex mac-transparent-titlebar
- The title bar transparency can be controlled by the value of the
+  The title bar transparency can be controlled by the value of the
 @code{mac-transparent-titlebar} frame parameter.  If it is @code{nil},
 which is the default then title bar uses system color for its
 background.  If it is non-@code{nil} then the frame background color is
 used instead.
 
+It is recommended that the theme applied on the frame with transparent
+title bar matches current macOS appearance.  That is, when macOS
+appearance is Light the theme should have a light colored background.
+Likewise, when macOS appearance is Dark the theme should have a dark
+colored background.
+
```

Comment:
Should we point to `mac-effective-appearance-change-hook` here?

Reply:
Good idea!